health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/RenderConditionalField.js

```
@@ -147,18 +147,18 @@
                 ...prev,
                 [cField?.bindTo]: prev?.[cField?.bindTo]
                   ? [
-                      ...prev?.[cField?.bindTo],
-                      {
-                        code: crypto.randomUUID(),
-                        name: "",
-                      },
-                    ]
+                    ...prev?.[cField?.bindTo],
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)